Dockerfile

```diff
@@ -2,14 +2,13 @@ FROM golang:1.24.4 AS build-env
 
 WORKDIR /app
 
-COPY go.mod .
-COPY go.sum .
+COPY go.mod go.sum ./
+COPY lambda ./lambda
+COPY internal ./internal
```

Comment:
putting these below the `RUN go mod download` will slightly optimise the build layers